osmaxx/core/templates/pages/about_us.html

```diff
@@ -0,0 +1,28 @@
+{% extends "osmaxx/base.html" %}
+{% load i18n %}
+
+{% block main %}
+<div class="container-row content">
+    <h2>About OSMaxx</h2>
+    <p>
+        OSMaxx (short for "OpenStreetMap arbitrary excerpt export") cuts out OpenStreetMap data, processes it to
+        geodata, converts it to typical GIS fileformats and offers those for download.
+    </p>
+    <h3>Useful links</h3>
+    <p>Can be found in our separate <a href="{% url 'pages:downloads' %}">downloads page</a></p>
+
+    <h3>Impressum</h3>
+    <p>
+        OSMaxx has been developed by the Geometalab at Hochschule Zürich.
+        Geometalab can be reached via email: <a href="mailto:geometalab@hsr.ch">geometalab@hsr.ch</a>.
+    </p><p>
+        <strong>Geometalab</strong><br />
+        HSR Hochschule für Technik Rapperswil<br />
+        Institute for Software<br />
+        c/o Stefan Keller<br />
+        Oberseestrasse 10<br />
+        Postfach 1475<br />
+        CH-8640 Rapperswil<br />
```

Comment:
Oops, not as liberal as I thought they'd be. [UWG Art. 3 "Unlautere Werbe- und Verkaufsmethoden und anderes widerrechtliches Verhalten"](https://www.admin.ch/opc/de/classified-compilation/19860391/index.html#a3):
> Unlauter handelt insbesondere, wer:
> [...]
> **s.** Waren, Werke oder Leistungen im elektronischen Geschäftsverkehr anbietet und es dabei unterlässt:
>
> 1. klare und vollständige Angaben über seine Identität und seine Kontaktadresse einschliesslich derjenigen der elektronischen Post zu machen,
> 2. auf die einzelnen technischen Schritte, die zu einem Vertragsabschluss führen, hinzuweisen,
> 3. angemessene technische Mittel zur Verfügung zu stellen, mit denen Eingabefehler vor Abgabe der Bestellung erkannt und korrigiert werden können,
> 4. die Bestellung des Kunden unverzüglich auf elektronischem Wege zu bestätigen;

We offer either "Werke" or "Leistungen" and even though the contract is not made online (so point 2 doesn't apply) we probably do so "im elektronischen Geschäftsverkehr".